### PR TITLE
Bookstore components setup

### PR DESCRIPTION
**Pull request by adamilare**
In this update, the following were implemented;
- Add basic components for the pages
- Style the webpage

### CODE REVIEW

File: src/routes/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const Categories = () => (
+  <>
+    <h1>Under construction</h1>
+  </>
+);
```
Comment (Danieldotcomcoder) [inline src/routes/Categories.jsx]: Kindly make sure the Categories page contains a Button with the text "Check status"

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @adamilare , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/routes/Categories.jsx (hunk 1 above)
Comment (adamilare) [reply src/routes/Categories.jsx]: Thank you for your review @Danieldotcomcoder .
I apologize for stating this; no button was required in the given requirements. Can you please guide me to the requirement? 

image

Comment (Danieldotcomcoder) [reply src/routes/Categories.jsx]: Hello, I am sorry for the inconvenience, can you see this line in the project requirements?
  
image

Comment (adamilare) [reply src/routes/Categories.jsx]: Not an inconvenience at all, just want to know where the requirement is coming from.
Is that from Reviewer's requirements, I still can't find that clause in the one I have access to?
[Activity reference](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v3_0.md)

Comment (Danieldotcomcoder) [reply src/routes/Categories.jsx]: There suppose to be project requirments for Bookstore: initialize project with components, The link you refered to is the sneak peak not the project requirments

Comment (adamilare) [reply src/routes/Categories.jsx]: Now I see my error!
Thank you

**Review by frank1738 — APPROVED** [review]
Hi @adamilare , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Organized code 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**